examples/standalone/runprgm/makefile

```
@@ -0,0 +1,16 @@
+# ----------------------------
+# Makefile Options
+# ----------------------------
+
+NAME = DEMO
+ICON = icon.png
+DESCRIPTION =
```

```suggestion
DESCRIPTION = "CE C Toolchain Demo"
```
For consistency with the other examples